Tests/SonarScanner.MSBuild.Tasks.UnitTest/GetAnalyzerSettingsTests.cs

```diff
@@ -243,8 +243,8 @@ public void ConfigExists_SettingsMerged()
             {
                 "c:\\original.should.be.preserved\\analyzer1.DLL",
                 "f:\\original.should.be.preserved\\analyzer3.dll",
-                "c:\\original.should.be.preserved\\SonarAnalyzer.Fake.DLL",
                 "c:\\SonarAnalyzer\\should.be.preserved.SomeAnalyzer.dll",
+                "c:\\should.be.removed\\SonarAnalyzer.Fake.DLL", // We consider all analyzers starting with 'SonarAnalyzer' as ours, this will be removed as a duplicate reference
```

Comment:
Can we test for case insensitivity? Windows doesn't care about case sensitivity, however generally preserves casing.